### ISSUE TITLE
[A11Y] - All Editorial View | Notifications Bell, All control functions should be moved to the top of the list instead of the bottom.

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12677]
## Issue Description
Difficult for user functionality.  Confusing when there is a long list of notifcations and the action buttons are on the bottom of the page.  

Comment that triggered the ticket:
“Suggest that options in notifications tab ‘mark new’ ‘mark read’ ‘delete’ be moved to top of page so that functions on this page are more intuitive, I initially thought that if I checked an item it would be marked as complete and go away but learned this was not the case when I scrolled down.” 

## How to reproduce / Current behavior
Login as any user and access the notification bell icon.  Observe that the Action buttons `Mark New`, `Mark Read` and `Delete` are at the bottom

## What application are you using?
 OJS, OMP or OPS version 3.5.0-4

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual, using screenreader.

## Action
Move the action buttons to the top of the list.  This will allow for more intuitive flow.

## Additional information

<img width="993" height="210" alt="Image" src="https://github.com/user-attachments/assets/8c09b9bd-f8ac-4492-82d4-ba4fea6dff3d" />
